### PR TITLE
TSK-3452 fix wso paging

### PR DESCRIPTION
**Pull request by ozhey**

### CODE REVIEW

File: plugins/workspace-one-uem/custom_actions.go [hunk 1]
```diff
@@ -25,16 +27,16 @@ func getDevicesByEmails(ctx *plugin.ActionContext, request *plugin.ExecuteAction
...
 		return nil, err
 	}
 
 	rawDeviceList = append(rawDeviceList, res.Devices...)
-
-	if res.Total > 1 {
-		for i := 1; i < res.Total; i++ {
-			res2, err := sendSearchDeviceRequest(ctx, request, plugin, i)
+	numOfPages := 1 + res.Total/pageSize
+	if numOfPages > 1 {
```
Comment (Idanraman) [inline plugins/workspace-one-uem/custom_actions.go:37]: How can this be false?

File: plugins/workspace-one-uem/custom_actions_test.go [hunk 2]
```diff
@@ -72,8 +71,8 @@ func (suite *WorkspaceOneUEMTestSuite) AddSearchDeviceEndpoint(router *gin.Engin
 			c.JSON(http.StatusBadRequest, gin.H{"error": "page number not provided"})
 			return
 		}
-		if pageSize == "" {
-			c.JSON(http.StatusBadRequest, gin.H{"error": "page size not provided"})
+		if pageSize != "500" {
```
Comment (Idanraman) [inline plugins/workspace-one-uem/custom_actions_test.go:74]: Lets use a const for 500 everywhere

File: plugins/workspace-one-uem/custom_actions_test.go [hunk 3]
```diff
@@ -90,15 +89,15 @@ func (suite *WorkspaceOneUEMTestSuite) AddSearchDeviceEndpoint(router *gin.Engin
...
+				Total:    501,
 			}
 		} else if page == "1" {
 			result = SearchDevicesResult{
 				Devices:  []Device{suite.device2},
 				Page:     1,
-				PageSize: 1,
-				Total:    2,
+				PageSize: 500,
+				Total:    501,
```
Comment (Idanraman) [inline plugins/workspace-one-uem/custom_actions_test.go:100]: Lets use a const for 500 everywhere

**Review by Idanraman — CHANGES_REQUESTED** [review]

File: plugins/workspace-one-uem/custom_actions_test.go (hunk 2 above)
Comment (ozhey) [reply plugins/workspace-one-uem/custom_actions_test.go:74]: Will do

File: plugins/workspace-one-uem/custom_actions_test.go (hunk 3 above)
Comment (ozhey) [reply plugins/workspace-one-uem/custom_actions_test.go:100]: Will do

File: plugins/workspace-one-uem/custom_actions.go (hunk 1 above)
Comment (ozhey) [reply plugins/workspace-one-uem/custom_actions.go:37]: Its bigger than one (not equal)
And integer division result is also integer
So 1 + 123 / 500 = 1

Comment (ozhey) [reply plugins/workspace-one-uem/custom_actions.go:37]: But it's redundant anyway, the for loop checks it too. Will remove

**Review by Idanraman — APPROVED** [review]

**Pull request merged**